src/components/ec-metroline-item/ec-metroline-item.vue

```diff
@@ -0,0 +1,203 @@
+          v-else
+          data-test="ec-metroline-item__status-disk-completed"
+          name="simple-check"
+          :size="14"
+        />
+      </div>
+
+      <div
+        v-if="!isLastItem"
+        data-test="ec-metroline-item__status-bar"
+        class="ec-metroline-item__status-bar"
+      />
+    </div>
+
+    <div
+      data-test="ec-metroline-item__content"
+      class="ec-metroline-item__content"
+    >
+      <div
+        data-test="ec-metroline-item__header"
+        class="ec-metroline-item__header"
+      >
+        <div
+          data-test="ec-metroline-item__header-heading"
+          class="ec-metroline-item__header-heading"
+        >
+          <slot name="heading" />
+        </div>
+
+        <div
+          v-if="isCompleted"
+          data-test="ec-metroline-item__header-sub-heading"
+          class="ec-metroline-item__header-sub-heading"
+        >
+          <slot name="sub-heading" />
+        </div>
+
+        <div
+          v-if="isCompleted"
+          data-test="ec-metroline-item__header-cta"
+        >
+          <slot name="header-cta" />
+        </div>
+      </div>
+
+      <div
+        v-if="!isNextItem"
+        class="ec-metroline-item__main"
+        data-test="ec-metroline-item__main"
+      >
+        <slot name="main" />
+      </div>
+
+      <div
+        v-if="!isNextItem"
+        class="ec-metroline-item__footer"
+        data-test="ec-metroline-item__footer"
+      >
+        <div
+          class="ec-metroline-item__footer-cta"
+          data-test="ec-metroline-item__footer-cta"
+        >
+          <slot name="footer-cta" />
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import EcIcon from '../ec-icon';
+import * as STATUS from '../../enums/metroline-status';
+
+export default {
+  name: 'EcMetrolineItem',
+  components: { EcIcon },
+  props: {
+    itemIndex: {
+      type: String,
+      required: true,
+    },
+    isLastItem: {
+      type: Boolean,
+      default: false,
+    },
+    status: {
+      type: String,
+      required: true,
+      validator(value) {
+        return [STATUS.NEXT, STATUS.ACTIVE, STATUS.COMPLETED].includes(value);
+      },
+    },
+  },
+  computed: {
+    isNextItem() {
+      return this.status === STATUS.NEXT;
+    },
+    isActive() {
+      return this.status === STATUS.ACTIVE;
+    },
+    isCompleted() {
+      return this.status === STATUS.COMPLETED;
+    },
+  },
+};
+</script>
+
+<style>
+.ec-metroline-item {
+  @apply tw-w-full;
+  @apply tw-flex;
+  @apply tw-mb-8;
+
+  &--is-last {
+    @apply tw-mb-0;
+  }
+
+  &__status {
+    @apply tw-flex tw-flex-col tw-items-center;
+    @apply tw-mr-16;
+  }
+
+  &__status-disk {
+    @apply tw-flex tw-justify-center tw-items-center;
+    @apply tw-bg-gray-6 tw-text-gray-3 tw-fill-current;
+    @apply tw-rounded-1/2;
+    @apply tw-min-w-24 tw-min-h-24;
+    @apply tw-small-text tw-text-center;
+    @apply tw-mb-8;
+
+    .ec-metroline-item--active & {
```

Comment:
I think we should not do this if we are using BEM. I mean, nested selectors. If we do this, it can no longer be considered BEM methodology.
Maybe we could just define another more specific class name, something like a "_ec-metroline-item__status-disk--active_" class, and then using it inside the "_ec-metroline-item__status-disk_" element: 
```
:class="{
    ec-metroline-item__status-disk--active': isActive || ( isCompleted && isLastItem)
}"
```

Reply:
Hey Jose, I thought we were ok with this selector that's why I use it here. We use it on several other components. If you think we can't use it here I can create individual active classes.

Reply:
yep, we violated the BEM with this on several places already. The theory says that we should have active modifier for each affected element, in practice, we use this shortcut even if know it's wrong.

Reply:
Ok, fine with me then :+1:

Reply:
Agree with the rest, this is not strictly BEM, but improves readability and we'll make the exception